src/style.css

```diff
@@ -0,0 +1,49 @@
+body {
+  margin: 0;
+}
+
+.stars__star {
+  display: flex;
+  background: url(./images/star.svg) no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
```

Comment:
Please note that the last star doesn't need a margin right

Hint: :last-child and :not 😎